spec/integration/basic_subscriber_spec.rb

```diff
@@ -27,6 +25,7 @@ def cancelled
       exchange = channel.topic("events")
       exchange.publish("Ohai Booked", :routing_key => "greg.basic_push.booked")
       exchange.publish("Ohai Cancelled", :routing_key => "basic.cancelled")
+      sleep 0.1
```

Comment:
I never ran into this problem locally with jruby-1.7.18 or jruby-9k, but on travis they were failing alot in a way where they didn't get the messages back from rabbitmq in this test. I think the issue was that it tried to pop right after the publish, but the publish is async so the messages weren't ready in the queues yet (I think)

So I added this sleep and now the travis tests seem to pass just fine...